roles/fio-distributed/tasks/main.yml

```diff
@@ -119,7 +119,7 @@
   - name: Create IP list
     set_fact:
       fio_hosts: |
-         {% for ip, node in pod_details.iteritems() %}
+         {% for ip, node in pod_details.items() %}
```

Comment:
nice catch, as iirc there's no iteritems() in python3, as items() itself does the same job

Reply:
just to keep in mind though is that, if ansible's using python2 then this might become expensive at a large scale ( which is unlikely to happen ) haven't seen someone run fio with >100s of pods yet